scriptSprint11.js

```diff
@@ -0,0 +1,42 @@
+const textAdvice = document.getElementById("advice-text");
```

Comment:
(opt)
```suggestion
const textAdviceElement = document.getElementById("advice-text");
```

Pentru ca folosesti aceasta variabile mai jos (fiind globala) - va fi mai util o denumire mai specifica, in cazul dat o sa stim ca e un element din HTML